docker/test-images/zipkin-eureka/src/main/java/zipkin/test/EurekaSecurity.java

```diff
@@ -33,7 +33,7 @@
 @Configuration
 @ConditionalOnProperty("eureka.username")
 @EnableConfigurationProperties(EurekaProperties.class)
-@ImportAutoConfiguration(SecurityAutoConfiguration.class)
+@Import(SecurityAutoConfiguration.class)
```

Comment:
So this one didn't work out? (just curious since we're importing auto configuration)

Reply:
yeah it threw the error I mentioned in the description. Then, I fought for an hour over CSRF. Finally, I axed this thing for a basic filter (pun intended)